src/System.Security.Cryptography.Algorithms/pkg/System.Security.Cryptography.Algorithms.pkgproj

```diff
@@ -5,9 +5,12 @@
     <ProjectReference Include="..\ref\4.0\System.Security.Cryptography.Algorithms.csproj">
       <SupportedFramework>net46</SupportedFramework>
     </ProjectReference>
-    <ProjectReference Include="..\ref\System.Security.Cryptography.Algorithms.csproj">
+    <ProjectReference Include="..\ref\4.1\System.Security.Cryptography.Algorithms.csproj">
       <SupportedFramework>net461;netcore50;netstandardapp1.5</SupportedFramework>
     </ProjectReference>
+    <ProjectReference Include="..\ref\System.Security.Cryptography.Algorithms.csproj">
+      <SupportedFramework>net463;netcore50;netstandardapp1.6</SupportedFramework>
+    </ProjectReference>
```

Comment:
For now remove netcore50 from here, netcore50 is only NS1.4 and won't be able to support this yet.  Also, netstandardapp1.6 doesn't exist.  We should switch to using netcoreapp1.0 for validation.